test/navigation/NavigationSpec.scala

```diff
@@ -519,7 +519,11 @@ class NavigationSpec extends SpecBase {
       }
 
       s"called with $UploadEvidenceQuestionPage" in new Setup {
-        val result: Call = mainNavigator.nextPage(UploadEvidenceQuestionPage, NormalMode, None)(fakeRequestWithCorrectKeysAndReasonableExcuseSet("other"))
+        val result: Call = mainNavigator.nextPage(UploadEvidenceQuestionPage, NormalMode, None)(fakeRequestConverter(fakeRequestWithCorrectKeys
+          .withSession(
+            SessionKeys.reasonableExcuse -> "other",
+            SessionKeys.isUploadEvidence -> "yes"
+          )))
         result.url shouldBe controllers.routes.OtherReasonController.onPageLoadForUploadEvidenceQuestion(NormalMode).url
```

Comment:
This doesn't seem right?

Reply:
Yes, removed the .withSession parameters.